00:00:00.000307 main     VirtualBox COM Server 6.1.16 r140961 win.amd64 (Oct 16 2020 16:33:17) release log
00:00:00.000310 main     Log opened 2020-11-23T15:52:32.259474400Z
00:00:00.000311 main     Build Type: release
00:00:00.000314 main     OS Product: Windows 10
00:00:00.000314 main     OS Release: 10.0.18363
00:00:00.000315 main     OS Service Pack: 
00:00:00.015617 main     DMI Product Name: 20L6S9U200
00:00:00.022793 main     DMI Product Version: ThinkPad T480
00:00:00.022817 main     Firmware type: BIOS
00:00:00.022829 main     Host RAM: 32426MB (31.6GB) total, 25921MB (25.3GB) available
00:00:00.022833 main     Executable: C:\Program Files\Oracle\VirtualBox\VBoxSVC.exe
00:00:00.022834 main     Process ID: 10548
00:00:00.022835 main     Package type: WINDOWS_64BITS_GENERIC
00:00:00.076853          VirtualBox: object creation starts
00:00:00.077067          Home directory: 'F:\PortableApps\Portable-VirtualBox\data\.VirtualBox'
00:00:00.077150          Installed Drivers:
00:00:00.079005            C:\WINDOWS\system32\DRIVERS\VBoxNetLwf.sys (Version: 6.1.16.40961)
00:00:00.079521            C:\WINDOWS\system32\DRIVERS\VBoxUSBMon.sys (Version: 6.1.16.40961)
00:00:00.079824            C:\WINDOWS\system32\DRIVERS\VBoxDrv.sys (Version: 6.1.16.40961)
00:00:00.080535            C:\WINDOWS\System32\drivers\VBoxNetAdp6.sys (Version: 6.1.16.40961)
00:00:00.085713          Loading settings file "F:\PortableApps\Portable-VirtualBox\data\.VirtualBox\VirtualBox.xml" with version "1.12-windows"
00:00:00.086670          usbLibGetDevices: Starting USB device enumeration
00:00:00.087877          Getting USB descriptor (id 0) failed (dwErr=31) on hub USB#ROOT_HUB30#4&d1fe901&0&0#{f18a0e88-c30c-11d0-8815-00a0c906bed8} port 3
00:00:00.087916          Getting USB descriptor (id 0) failed (dwErr=31) on hub USB#ROOT_HUB30#4&d1fe901&0&0#{f18a0e88-c30c-11d0-8815-00a0c906bed8} port 7
00:00:00.087953          Getting USB descriptor (id 0) failed (dwErr=31) on hub USB#ROOT_HUB30#4&d1fe901&0&0#{f18a0e88-c30c-11d0-8815-00a0c906bed8} port 8
00:00:00.087986          Getting USB descriptor (id 0) failed (dwErr=31) on hub USB#ROOT_HUB30#4&d1fe901&0&0#{f18a0e88-c30c-11d0-8815-00a0c906bed8} port 9
00:00:00.088908          Getting USB descriptor (id 0) failed (dwErr=31) on hub USB#ROOT_HUB30#4&d1fe901&0&0#{f18a0e88-c30c-11d0-8815-00a0c906bed8} port 15
00:00:00.089037          Getting USB descriptor (id 0) failed (dwErr=31) on hub USB#ROOT_HUB30#5&d2d70c4&0&0#{f18a0e88-c30c-11d0-8815-00a0c906bed8} port 1
00:00:00.089079          Getting USB descriptor (id 0) failed (dwErr=31) on hub USB#ROOT_HUB30#5&d2d70c4&0&0#{f18a0e88-c30c-11d0-8815-00a0c906bed8} port 2
00:00:00.089116          Getting USB descriptor (id 0) failed (dwErr=31) on hub USB#ROOT_HUB30#5&d2d70c4&0&0#{f18a0e88-c30c-11d0-8815-00a0c906bed8} port 3
00:00:00.090725          usbLibGetDevices: Found 10 USB devices, 0 captured
00:00:00.090802          USB: Unknown USB device detected (idVendor: 0x06cb, idProduct: 0x009a)
00:00:00.090814          USB: Unknown USB device detected (idVendor: 0x0bda, idProduct: 0x0316)
00:00:00.090826          USB: Unknown USB device detected (idVendor: 0x13d3, idProduct: 0x56a6)
00:00:00.090842          USB: Unknown USB device detected (idVendor: 0x8087, idProduct: 0x0a2b)
00:00:00.090851          USB: Unknown USB device detected (idVendor: 0x8087, idProduct: 0x0ac9)
00:00:00.090864          USB: Unknown USB device detected (idVendor: 0x8087, idProduct: 0x0aca)
00:00:00.090871          USB: Unknown USB device detected (idVendor: 0x8087, idProduct: 0x0af1)
00:00:00.175193          HostDnsMonitor: initializing
00:00:00.182956          HostDnsMonitor: updating information
00:00:00.182975          HostDnsMonitor: old information
00:00:00.182976            no server entries
00:00:00.182977            no domain set
00:00:00.182977            no search string entries
00:00:00.182978          HostDnsMonitor: new information
00:00:00.182978            server 1: 127.0.0.1
00:00:00.182979            domain: ni.corp.natinst.com
00:00:00.182979            no search string entries
00:00:00.280829          VD: VDInit finished with VINF_SUCCESS
00:00:00.283401          Loading settings file "F:\PortableApps\Portable-VirtualBox\data\.VirtualBox\Machines\LinuxLive\LinuxLive.vbox" with version "1.14-windows"
00:00:00.286296          Loading settings file "F:\PortableApps\Portable-VirtualBox\data\.VirtualBox\Machines\Debug\Debug.vbox" with version "1.14-windows"
00:00:00.296349          OCI: Local config file 'F:\PortableApps\Portable-VirtualBox\data\.VirtualBox\oci_config' does not exist
00:00:00.296454          OCI: Original config file 'C:\Users\inthirua\.oci\config' does not exist
00:00:00.296455          OCI: Reading profiles finished with status ERROR_SUCCESS
00:00:00.296461          ExtPack: Created cloud provider 'OCI' (hrc=ERROR_SUCCESS)
00:00:00.296483          VirtualBox: object created
00:00:00.775778          Launched VM: 73321120 pid: 14996 (0x3a94) frontend: GUI/Qt name: LinuxLive
00:00:04.374348          Load [C:\Program Files\Oracle\VirtualBox\ExtensionPacks\Oracle_VM_VirtualBox_Extension_Pack\win.amd64\VBoxHostWebcam.DLL] vrc=VINF_SUCCESS
00:00:04.611052          ERROR [COM]: aRC=VBOX_E_IPRT_ERROR (0x80bb0005) aIID={85632c68-b5bb-4316-a900-5eb28d3413df} aComponent={SessionMachine} aText={Saved screenshot data is not available (VERR_NOT_SUPPORTED)}, preserve=false aResultDetail=-37
00:00:54.850059          Saving settings file "F:\PortableApps\Portable-VirtualBox\data\.VirtualBox\Machines\LinuxLive\LinuxLive.vbox" with version "1.14-windows"
00:01:01.845771 main     VirtualBox: object deletion starts
00:01:02.786211 main     ERROR [COM]: aRC=VBOX_E_OBJECT_IN_USE (0x80bb000c) aIID={ad47ad09-787b-44ab-b343-a082a3f2dfb1} aComponent={MediumWrap} aText={Medium 'F:\PortableApps\Portable-VirtualBox\data\.VirtualBox\Machines\LinuxLive\HardDisks/LinuxLive.vmdk' cannot be closed because it is still attached to 2 virtual machines}, preserve=false aResultDetail=0
00:01:02.793899 main     HostDnsMonitor: shutting down ...
00:01:02.799323 main     HostDnsMonitor: shut down
00:01:02.812765 Watcher  ERROR [COM]: aRC=E_ACCESSDENIED (0x80070005) aIID={d0a0163f-e254-4e5b-a1f2-011cf991c38d} aComponent={VirtualBoxWrap} aText={The object is not ready}, preserve=false aResultDetail=0
00:01:02.818204 main     VirtualBox: object deleted
